core/src/test/java/org/infinispan/xsite/offline/AsyncOfflineTest.java

```diff
@@ -37,6 +38,7 @@
  * @author Pedro Ruivo
  * @since 10.0
  */
+@CleanupAfterMethod
```

Comment:
Looks like this needs another JIRA as well?

Reply:
Ooops, yeah. I'll omit this one for now since I'll probably come back to the test suite later,